docs/source/user_guide/gcm_based_inference/answering_causal_questions/computing_counterfactuals.rst

```diff
@@ -26,7 +29,8 @@ To see how the method works, let's generate some data:
 >>> Z = 3*Y + np.random.normal(loc=0, scale=1, size=1000)
 >>> training_data = pd.DataFrame(data=dict(X=X, Y=Y, Z=Z))
 
-Next, we'll model cause-effect relationships as an invertible SCM and fit it to the data:
+Next, we'll model cause-effect relationships and fit the models to the data. Estimating counterfactuals requires an
```

Comment:
1. The data generation process can be made more realistic. Currently, there can be negative dosages too (absurd!).
2. Should we also explain what Y is?

Reply:
I completely rewrote the example to keep it connected to the cholesterol example while being somewhat 'realistic'.